custom/icds_reports/utils/aggregation_helpers/distributed/migration_forms.py

```diff
@@ -29,7 +29,16 @@ def data_from_ucr_query(self):
             supervisor_id,
             %(month)s::DATE AS month,
             person_case_id AS person_case_id,
-            LAST_VALUES(is_migrated) OVER w AS is_migrated,
+            LAST_VALUES(is_migrated) OVER w AS migration_status,
+            CASE
+                WHEN LAST_VALUES(date_left) OVER w IS NULL
+                    THEN LAST_VALUES(timeend)
+                ELSE LAST_VALUES(date_left) OVER w
+            END AS migration_date,
+            CASE
```

Comment:
you don't have to have this column. Only plain one `LAST_VALUES(is_migrated) OVER w AS migration_status,` is fine.

This check we can do in following agg tables.